pymobiledevice3/services/crash_reports.py

```diff
@@ -138,10 +138,15 @@ def get_new_sysdiagnose(self, out: str, erase: bool = True) -> None:
         self._wait_for_sysdiagnose_to_finish()
         self.pull(out, entry=sysdiagnose_filename, erase=erase)
 
+    @staticmethod
+    def _sysdiagnose_complete_syslog_match(message: str) -> bool:
+        message = message.lower()
+        return message == 'sysdiagnose (full) complete' or 'sysdiagnose completed' in message
```

Comment:
going by the syslog message structure in your first provided message, the message should actually be with a capital `S` in here. the test was actually correct

Reply:
Will update, based on your recommendation.